src/common/LogClient.cc

```diff
@@ -133,6 +133,10 @@ LogClient::LogClient(CephContext *cct, Messenger *m, MonMap *mm,
 {
 }
 
+void LogChannel::set_log_to_monitors(bool v) {
```

Comment:
@@gerald-yang could you put the opening bracket "{" on a new line so it is consistent with other part of this source. also it improves the readability.